src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -113,9 +119,9 @@ public void start(Map<String, String> properties) {
       switch (queryMode) {
         case TABLE:
           for (String table : tables) {
-            Map<String, String> partition =
-                Collections.singletonMap(JdbcSourceConnectorConstants.TABLE_NAME_KEY, table);
-            partitions.add(partition);
+            List<Map<String, String>> tablePartitions = possibleTablePartitions(table);
```

Comment:
I think we need a comment here that describes what we're doing:

     // Find possible partition maps for different offset protocols ...